apps/www/components/Quote.tsx

```diff
@@ -11,12 +11,17 @@ export default function Quote(props: Props) {
   const { children, caption, img } = props
 
   return (
-    <figure className="border border-gray-100 p-8 dark:border-gray-600">
-      <div className="flex flex-col gap-5">
-        <blockquote className="text-scale-1200 mb-4 border-none" style={{ margin: 0, padding: 0 }}>
+    <figure className="quote-figure p-0 mt-8">
```

Comment:
ahh i should of maybe said before...
you can write some (weird) tailwind for this that only me and you will understand 😅:

`![&_p:first-child]:mt-0 ![&_p:last-child]:mt-0 `

also (sorry if you already know this..) if your just trying to remove the margins added by 'prose', you can wrap the whole component in `.not-prose` to stop tailwind typography applying css in child elements.